fuel/app/classes/materia/session/play.php

```diff
@@ -65,7 +65,7 @@ public function start($user_id=0, $inst_id=0, $context_id=false, $is_preview=fal
 			// Essentially true but fragile.
 			$is_lti = array_key_exists('lti_message_type', $this->environment_data['input']) || array_key_exists('token', $this->environment_data['input']);
 			$this->auth = $is_lti ? 'lti' : '';
-			$this->referrer_url = \Input::referrer();
+			$this->referrer_url = mb_strimwidth(\Input::referrer(), 0, 255);
```

Comment:
The referrer url can be longer than the database will handle, this will limit it.